bundles/org.openhab.binding.wundergroundupdatereceiver/README.md

```diff
@@ -0,0 +1,115 @@
+# Wunderground Update Receiver Binding
+
+Many personal weather stations are only capable of submitting measurements to the wunderground.com update site.
+
+This binding enables acting as a receiver of updates from weather stations that post measurements
+to https://rtupdate.wunderground.com/weatherstation/updateweatherstation.php. If the hostname is configurable - as on
+weather stations based on the Fine Offset Electronics WH2600-IP - this is simple, otherwise you have to set up dns
+such that it resolves the above hostname to your server, without preventing the server from resolving the proper ip if
+you want to forward the request.
+
+The server thus listens at http(s)://<your-openHAB-server>:<openHAB-port>/weatherstation/updateweatherstation.php and the weather station
+needs to be pointed at this address. If you can't configure the weather station itself to submit to an alternate hostname you would need
+to set up a dns server that resolves rtupdate.wunderground.com to the IP-address of your server and provide as dns to the weather station
+does DHCP. Make sure not to use this dns server instance for any other DHCP clients. 
+
+The request is in itself simple to parse, so by redirecting it to your openhab server you can intercept the values and use them to
+control items in your home. E.g. use measured wind-speed to close an awning or turn on the sprinkler system after some time without rain.
+This binding allows you to mix and match products from various manufacturers that otherwise have a closed system.
+
+If you wish to pass the measurements on to rtupdate.wunderground.com, you can use a simple rule that triggers on the
+wundergroundupdatereceiver:wundergroundUpdateReceiver:<channel-id>:metadata#last-query-trigger to do so. It can also be used to
+submit the same measurements to multiple weather services via multiple rules.
```

Comment:
```suggestion
wundergroundupdatereceiver:wundergroundUpdateReceiver:<channel-id>:metadata#last-query-trigger to do so.
It can also be used to submit the same measurements to multiple weather services via multiple rules.
```

Reply:
It's part of the same paragraph. It should not be seperated.

Reply:
It will be rendererd in the same paragraph in the final document, although if the sentences are in separate lines in markdown. Each sentence should be in its own line to make reading diffs easier.

Reply:
I see. It makes editing the document more difficult though, as prose is less compact than code and results in some very long side-scrolled lines.